predicate/cli/tests/data/buggy_policy.py

```diff
@@ -0,0 +1,37 @@
+from solver.ast import Duration
+from solver.teleport import AccessNode, Node, Options, OptionsSet, Policy, Rules, User, JoinSession
+
+
+class Developer:
+    p = Policy(
+        name="developer",
+        loud=False,
+        allow=Rules(
+            AccessNode(
+                ((AccessNode.login == "testuser") & (User.name != "testuser"))
+                | (User.traits["team"] == ("testteam",))
+            ),
+            Rules(
```

Comment:
Is this faulty rules node supposed to be here for the test functionality?

Reply:
Yup this one is to test faulty rule.